app/notifiers/course/assessment_notifier.rb

```diff
@@ -5,4 +5,15 @@ def assessment_attempted(user, assessment)
       notify(assessment.tab.category.course, :feed).
       save
   end
+
+  # To be called when user submitted an assessment.
+  def assessment_submitted(user, course_user, submission)
+    group_managers = course_user.groups.flat_map { |g| g.group_users.manager }.map(&:course_user)
```

Comment:
Can you add a TODO here, we should actually provide `course_user#group_managers` from the Groups and GroupUser model to calculate this rather than having to calculate it when we require.